ee/codegen/src/serializers/vellum.ts

```diff
@@ -14,6 +14,7 @@ import {
   Schema,
   record as recordSchema,
   unknown as unknownSchema,
+  union,
```

Comment:
all other changes stay confined to this file, without polluting the types we care about